source/server/event_publisher.js

```diff
@@ -0,0 +1,139 @@
+Space.Object.extend(Space.jobQueue, 'EventPublisher', {
+
+  dependencies: {
+    jobCollection: 'Space.jobQueue.Jobs'
+  },
+
+  onDependenciesReady() {
+    var self = this;
+    this.jobCollection.events.on('call', function (message) {
+      switch(message.method) {
+        case 'startJobServer':
+          self._onStartJobServer(message);
```

Comment:
i would just publish the events directly in the switch statement – the methods don't add much value here